src/uwtools/config/formats/base.py

```diff
@@ -36,10 +32,10 @@ def __init__(self, config: Optional[Union[dict, str, Path]] = None) -> None:
         else:
             self._config_file = str2path(config) if config else None
             self.data = self._load(self._config_file)
-        if self.get_depth_threshold() and self.depth != self.get_depth_threshold():
+        if self._get_depth_threshold() and self._depth != self._get_depth_threshold():
```

Comment:
This is slightly awkward, as elsewhere where either this or another of the methods now marked private is accessed a `pylint: disable=protected-access` suppressor is now needed, but I did not like seeing these irrelevant-to-end-users methods appearing in the RST docs. I think the ends (the final RST documentation) justifies the means, but it's too bad that we must (I think) make some internal compromises for the best end-user experience.

Reply:
Thats an interesting point - now that you mention it, I'm surprised we actually left these public at first, but a necessary sacrifice